includes/push-ui.php

```diff
@@ -67,6 +68,126 @@ function syndicatable() {
 	return true;
 }
 
+/**
+ * Get available connections for use in the Push UI.
+ *
+ * @return void
+ */
+function get_connections() {
+	if ( ! check_ajax_referer( 'dt-load-connections', 'loadConnectionsNonce', false ) ) {
+		wp_send_json_error();
+		exit;
```

Comment:
I think  there is no need for exit after wp_send_json_*, they die or wp_die.

Reply:
Yeah, good catch. Copied this from the other `ajax` function we have here but these aren't needed.